lib/tasks/db.rake

```diff
@@ -0,0 +1,10 @@
+namespace :db do
+  desc "Recreate public schema."
```

Comment:
only doing this b/c I don't have a full DLE instance for this sample application.  Just using a fully reset version of the demo DLE instance